app/models/shelter.rb

```diff
@@ -31,4 +33,23 @@ def alphabetical_pets
   def shelter_pets_filtered_by_age(age_filter)
     adoptable_pets.where("age >= ?", age_filter)
   end
+
+  def self.reverse_order
+    find_by_sql("SELECT * FROM shelters ORDER BY shelters.name desc;")
+  end
+
+  def self.shelters_pending_applications
+   Shelter.joins(pets: :applications).where(applications: {status:"Pending"}).pluck(:name)
+  end
```

Comment:
Great data querying in these model methods! I see you wrote this join in raw SQL below. Great practice! Did you get it working? I've only reviewed it briefly but it looks great to me